src/pages/Product/ProductList/Product/ProductInfo/ProductInfo.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../ProductInfo/ProductInfo.scss';
+
+function ProductInfo({ product }) {
+  return (
+    <Link to="/category" className="goToDetail">
```

Comment:
```suggestion
function ProductInfo({ product }) {
  return (
    <Link to="/category" className="productInfo">
```

- CRA로는 index.html이 하나인 SPA프로젝트를 구성하는 것이고 이런 상황에서 결과적으로 모든 css파일들은 하나의 html에 한번에 불러와집니다.
- 결과적으로 모든 css파일이 한번에 불러와지기에 이 css파일에서 부여한 스타일이 동일한 className을 가진 다른 컴포넌트의 요소들에게도 영향을 미칠 수 있습니다.
- 따라서 반드시 **최상위 태그의 className을 컴포넌트 이름과 동일하게 부여한 뒤 그 className하위로 네스팅** 적용해주세요
- 다른 컴포넌트도 수정 해 주세요
```javascript
const ProductDetail = () => {

	return(
		<section className="productDetail">
			<h1 className="title">Title<h1>
		</section>
	)
}
```

Reply:
해당 리뷰사항 ProductList내 폴더에 있는 모든 파일에 반영했습니다! 감사합니다!